src/_provisional.scss

```diff
@@ -149,12 +149,110 @@
 
 }
 
+/*This is a rehash of another selector... the point of it is to allow the "force-style-gcweb-latest" class to be placed directly on a <form> element. Can't do it more elegantly since elements can't come after classes in selector rules.*/
+form {
+	&.provisional {
+		&.force-style-gcweb-latest {
+			@extend %gcweb-default-font-size;
+			color: green;
+		}
+	}
+}
+
 .provisional {
 	//This color will eventually be replaced by the new colors from the alert revamp PR.
 	&.icon-warning-light, .icon-warning-light {
 		color: #DF7200;
 	}
 
+	/* Force form elements to always use GCWeb's latest font styles
+	 * Note: This is an evergreen class. Its styles will change to adapt to any future GCWeb font updates.
+	 */
+	&.force-style-gcweb-latest {
+		form,
+		.checkbox,
+		.checkbox-inline,
+		.radio,
+		.radio-inline,
+		.btn:not(.btn-call-to-action) {
+			@extend %gcweb-default-font-size;
+		}
+
+		&.btn:not(.btn-call-to-action), &.form-control,
+		.btn:not(.btn-call-to-action), .form-control {
+			padding: 6px 12px;
+			/*TODO: This conflicts with the below selector that's based on the call to action button's paddinge*/
+		}
+
+		/*Ripoff of call to action button properties to use as baseline for normal buttons*/
```

Comment:
@EricDunsworth Wouldn't it not be better to the `btn-call-to-action` to behave more like WET/Bootstrap button? Maybe resizing the WET/Bootstrap buttons to match the `btn-call-to-action` font-size would prevent conflicts and tons of overrides.

Reply:
I'm currently experimenting a bit more with button font sizes/padding/etc to see how that might work out in practice. Like how things would be like if call to action didn't exist.

It's trickier than just making each button size use the next highest size's properties though. Bootstrap uses a sliding scale of sorts when increasing font size, padding, etc for bigger button sizes. So it isn't as straightforward as adding like 4px extra to each property for each size increment. I think part of it will be a matter of perception/opinion.

What I find weirdest about call to action buttons is that their font size is 22px... GCWeb's baseline font is 20px (WET is 16px, Bootstrap is 14px). Both WET and Bootstrap re-use their baseline font size for buttons (so normal button text isn't any larger in those).

The one thing about call to action buttons that's kind of grown onto me is text-shadow... it gives off the perception of slightly higher contrast around white text. So I might sneak it into normal buttons that use white text. Unfortunately it doesn't really work out for darker text though (causes the text to get kind of blurry).

Reply:
Yes, I've seen sliding scale in the wild recently. Do you think that we could leverage the Bootstrap sliding all the way up to GCWeb? I would much prefer to architect the buttons in GCWeb in a way that we can recreate `btn-call-to-action` from a common architecture, even if at end some design elements are exclusive the CTA button.

Reply:
Whoops - meant to say curve instead of sliding scale :S. What I had in mind was how Bootstrap doesn't seem to increment fonts/padding/etc by a preset amount when going from extra-small to large buttons. Which makes increasing button sizes trickier than it'd of otherwise been. For example, Bootstrap's ``btn-xs`` class uses 1px top/bottom padding, then increases to 5px for ``btn-sm``, 6px for ``btn`` and 10px for ``btn-lg``. There isn't really a preset pattern for how the increments are done. So I have to add my opinion into the mix to try determining what looks best in practice.

Haven't forced-pushed an update yet, but what I'm currently looking into is using ``@include button-size()`` to redeclare all button styles to what I'd consider to be appropriate for GCWeb. For call to action buttons, I'm undoing the non-standard properties (like margins/padding) to basically turn it into a colour class. Also using ``text-shadow`` on all buttons that contain white text (borrowed from call to action).

All of those alternations will require the ``provisional`` class to be used in some way or another. Either directly alongside the current element's classes (e.g. ``<button class="provisional btn btn-default">``) or alongside another provisional class on a parent element (e.g. ``<div class="provisional force-style-gcweb-latest">``).

Reply:
I was thinking of [curve](https://type-scale.com/) too. I'll wait for your push. For what I understand you're going in the direction that makes total sense.